monai/networks/blocks/warp.py

```diff
@@ -111,3 +111,38 @@ def forward(self, image: torch.Tensor, ddf: torch.Tensor) -> torch.Tensor:
             )
 
         return warped_image
+
+
+class DVF2DDF(nn.Module):
+    """
+    Layer calculates DVF from DDF with scaling and squaring.
```

Comment:
Just to make it a little easier for those not as familiar with registration it would be nice to state what DVF and DDF is here and what the significance of this layer is.